rcmd/Rsettings_test.go

```diff
@@ -36,3 +40,53 @@ func TestLibPathsEnv(t *testing.T) {
 		}
 	}
 }
+
+func TestGetRVersion(t *testing.T) {
+	var rVersionTests = []struct {
+		version    cran.RVersion
+		platform   string
```

Comment:
First, we could refactor out the parsing of information to a separate function that can be tested in isolation of the actual runtime

something like
```
version, platform := parseRMetadata([]string)
```

then instead of having each one of these as individual elements of a struct, you could do a slice of strings or a single string with line breaks to more accurately represent how data gets fed in